packages/app/src/providers/api.ts

```diff
@@ -35,12 +34,12 @@ export abstract class DataProviderApi {
    * - `() => Promise<Blob>`  Export is generated client-side
    * - `undefined`            Export scenario is not supported
    */
-  public getExportURL?<D extends Dataset<ArrayShape>>( // optional, so can't be abstract
```

Comment:
Removing the arbitrary restriction on `ArrayShape` to support exporting scalar datasets like scalar compounds.